kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -148,7 +145,7 @@
           }
           if (activityType) {
             if (activityType.value !== 'all') {
-              downloadsToDisplay = downloadsToDisplay.filter(download =>
+              downloadsToDisplay = this.downloadsToDisplay.filter(download =>
```

Comment:
This seems like an error - `this` does not refer to the component inside the `setup` context, as it is executed before the component VM is created.